integration/fuse/bin/alluxio-fuse-sdk

```diff
@@ -0,0 +1,476 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+SCRIPT_DIR="$(cd "$(dirname "$(readlink "$0" || echo "$0")")"; pwd)"
+
+readonly USAGE="Usage:\n\talluxio-fuse-sdk [mount|umount]
+
+Mount a UFS address to local mount point.
+Usage:\n\talluxio-fuse-sdk mount ufs_address mount_point [options]
+
+General options
+\t-h \tprint help
+\t-f \tFUSE foreground option - do not run as daemon
+
+Alluxio mount options
+\t-o <ALLUXIO_PROPERTY_KEY>=<ALLUXIO_PROPERTY_VALUE> \tThe credentials of the target ufs address should be provided here so that alluxio fuse can access the target data set. Other alluxio common/user configuration can also be provided here
+\t-o data_cache=<local_cache_directory> (Default=\"\" which means disabled) \tLocal folder to use for local data cache.
+\t-o data_cache_size=<size> (Default=\"512MB\") \tMaximum cache size for local data cache directory.  
+\t-o metadata_cache_size=<size> (Default=\"0\" which means disabled) \tMaximum number of entries in the metadata cache. Each 1000 entries cause about 2MB memory.
+\t-o metadata_cache_expire=<timeout> (Default=\"10min\") \tSpecify expire time for entries in the metadata cache
+
+FUSE mount options
+\tMost of the generic mount options described in 'man mount' and many FUSE specific mount options are supported.
+\tSome examples are listed below:
+\t-o direct_io \tDisables the use of page cache (file content cache) in the kernel for this filesystem.
+\t-o attr_timeout=<timeout_in_seconds> \tThe timeout in seconds for which file/directory attributes (as returned by e.g. the getattr handler) are cached.
+\t-o entry_timeout=<timeout_in_seconds> \tThe timeout in seconds for which name lookups will be cached.
+
+JVM options
+\tJVM options can be directly passed in to the script and will be directly used to launch the Fuse process
+\te.g. -Xms4g -Xmx4g -XX:MaxDirectMemorySize=4g
+
+Examples
```

Comment:
Is this only for the sdk fuse? How about hardcode the config items?
alluxio.user.ufs.address=true ?

Then the user doesn't bother to modify config file

Reply:
Those (ufs enabled and root ufs address) are hard-coded by AlluxioFuse, so users do not need to modify config file.
User can pass in the ufs credentials via command line java options or via config file